test/integration/targets/dms_endpoint/tasks/main.yml

```diff
@@ -0,0 +1,135 @@
+---
+
+- name: set connection information for all tasks
+  set_fact:
+    aws_connection_info: &aws_connection_info
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      #security_token: "{{ security_token }}"
+      region: "{{ aws_region }}"
+  no_log: yes
+
```

Comment:
May want to create a fact so the endpointidentifier will always be unique.


- name: generate unique value for testing
  set_fact:
    dms_identifier: "{{ resource_prefix }}-dms"